agent/SharkserversBuyer.py

```diff
@@ -0,0 +1,192 @@
+from random import randint
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import TimeoutException
+
+from BogusFormBuilder import BogusFormBuilder
+
+from VPSBuyer import VPSBuyer
+
+from Wallet import Wallet
+
+import selenium.webdriver.support.ui as ui
+
+import time
+
+from CountryGetter import CountryGetter
+
+
+class SharkserversBuyer(VPSBuyer):
+    '''
+    This class orders a VPS from sharkservers.co.uk
+    '''
+    def __init__(self, email = "", password = "", SSHPassword = ""):
+        super(SharkserversBuyer, self).__init__(email, password, SSHPassword)
+        self.email = email
+        if self.email == "":
+            self.email = self.generator.getEmail()
+            
+        if self.password != password or self.password == "":
+            self.password = self.generator.getRAString(32) + "!"
+            
+        if self.SSHPassword != SSHPassword or self.SSHPassword == "":
+            self.SSHPassword = self.generator.getRAString(32) # Generate new password
+            
+        self.SSHUsername = "root"
+        pass
+

```

Comment:
Unnecessary pass.

In addition please look at the code for the super.init() that is being used here. If you do you will see that this method contains large chunks of duplicate code. The check on line 32 also makes very little sense as self.SSHPassword always equals SSHPassword or SSHPassword is zero.